Multisig/UI/Settings/OwnerKeyManagement/Backup/UIFlow.swift

```diff
@@ -71,7 +71,13 @@ class UIFlow: NSObject {
     }
 
     func stop(success: Bool) {
-        if let presenter = presenter {
+        if let presenter = presenter, presenter.presentedViewController != nil {
```

Comment:
Is it possible that presenter != nil and  presenter.presentedViewController = nil ?

Reply:
yes, that's exactly the root cause of the view controller not dismissed. The view controller was presented from the child view controller, which forwards the presentation to the ancestor - main tab bar view controller. Calling `presenter.dismiss()` in that situation has no effect, because the presenter didn't have any controller presented.

This is not evident from the code, so I added another comment in the `else if` branch